### PR TITLE
Added Badges in Readme.md

### LINKED ISSUE #62: Add badges to the repo
Add appropriate badges to the repo for an overall good project. 
Task:
- research different badges(services to be integrated) that can be applied. Check which of these can be implemented right away. Discuss on this issue's comments section with contributors to finalise the badges.
Example of badges:
Capture1
Capture2

### PR DESCRIPTION
**Pull request by arghac14**
Fixes #62 
@coderwassananmol @Parikshit-Hooda I have added the badges, please review the PR! Thanks

### CODE REVIEW

**Comment by rajat2502** [comment]
LGTM

**Comment by arghac14** [comment]
@rajat2502 @coderwassananmol @Parikshit-Hooda can you please add 'gssoc20' and 'medium' (according to issue) labels on this PR, the organisers said it's important for the score.

File: README.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
 
 A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
 
-## Getting Started
+![](https://img.shields.io/github/repo-size/coderwassananmol/BUB) ![](https://img.shields.io/github/license/coderwassananmol/BUB2?color=red)<br>
+![](https://img.shields.io/github/issues/coderwassananmol/BUB2?color=green) ![](https://img.shields.io/github/issues-pr/coderwassananmol/BUB2?color=green) ![](https://img.shields.io/github/downloads/coderwassananmol/BUB2/total)  ![](https://img.shields.io/github/last-commit/coderwassananmol/BUB2) ![](https://img.shields.io/github/contributors/coderwassananmol/BUB2)<br><br>
+![](https://img.shields.io/github/forks/coderwassananmol/BUB2?style=social) ![](https://img.shields.io/github/stars/coderwassananmol/BUB2?style=social) ![](https://img.shields.io/github/watchers/coderwassananmol/BUB2?style=social)
+
+## Getting Started [![Generic badge](https://img.shields.io/badge/Get-started-<green>.svg)](https://github.com/coderwassananmol/BUB2)
```
Comment (rkpattnaik780) [inline README.md]: A badge looks unnecessary here.

File: README.md [hunk 2]
```diff
@@ -2,13 +2,17 @@
 
 A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs. Check out [BUB2 on Toolforge](https://bub2.toolforge.org)!
 
-## Getting Started
+![](https://img.shields.io/github/repo-size/coderwassananmol/BUB) ![](https://img.shields.io/github/license/coderwassananmol/BUB2?color=red)<br>
```
Comment (rkpattnaik780) [inline README.md]: Can you move the fork/stars/watch badges to right of the BUB2 heading?
And rest of the badges just below the heading? Above "A book uploader bot ..."

File: README.md [hunk 3]
```diff
@@ -2,13 +2,17 @@
...
+![](https://img.shields.io/github/forks/coderwassananmol/BUB2?style=social) ![](https://img.shields.io/github/stars/coderwassananmol/BUB2?style=social) ![](https://img.shields.io/github/watchers/coderwassananmol/BUB2?style=social)
+
+## Getting Started [![Generic badge](https://img.shields.io/badge/Get-started-<green>.svg)](https://github.com/coderwassananmol/BUB2)
 
 These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
 
 ### Prerequisites
 
-* [Node.JS](https://nodejs.org/en/download/) ( check for installation with ```npm -v``` and ```node -v``` on terminal )
+* [Node.JS](https://nodejs.org/en/download/) ( check for installation with ```npm -v``` and ```node -v``` on terminal ) ![](https://img.shields.io/npm/v/npm)
```
Comment (rkpattnaik780) [inline README.md]: Remove this badge as well.

**Comment by arghac14** [comment]
@rkpattnaik780 I have made the changes as requested, have a look!

**Comment by arghac14** [comment]
@rkpattnaik780 @rajat2502 @coderwassananmol @Parikshit-Hooda someone please add 'gssoc20' and 'medium' label to this PR, otherwise my score will not be counted towards gssoc;

**Pull request merged**